src/GafferImage/ImageWriter.cpp

```diff
@@ -1320,6 +1400,12 @@ ImageWriter::ImageWriter( const std::string &name )
 
 	colorSpaceChild->outputSpacePlug()->setValue( "${__imageWriter:colorSpace}" );
 
+	ValuePlugPtr namingConventionPlug = new ValuePlug( "namingConvention" );
```

Comment:
I feel like there may be a better name for this, since we're talking about part structure as much as naming, but I'm a bit short of ideas. `partStructure`? Or should we just call it `channelNaming` to match `Reader.channelNaming`?